server/testSubmission2.json

```diff
@@ -0,0 +1,15 @@
+{
+  "isSubmitted": false,
```

Comment:
dont forget to update the schema after you rename isSubmitted and removedFromJudging.